doc/src/manual/modules.md

```diff
@@ -122,6 +122,16 @@ There are three important standard modules:
 * [`Base`](@ref) contains basic functionality that is useful in almost all cases.
 * [`Main`](@ref) is the top-level module and the current module, when Julia is started.
 
+
+!!! note "Standard library modules"
+    By default Julia ships with some standard library modules. These behave like regular
+    Julia packages except that you don't need to install them explicitly. For example,
+    if you wanted to perform some unit testing, you could load the `Test` standard library
+    as follows:
```

Comment:
This is slightly misleading since in package you do need to "install" them by adding them with the package manager.